RELEASE_PROCESS.md

```diff
@@ -44,25 +44,41 @@ In order to release, you will need the following accounts/permissions:
 
 ### Testing the Release
 
-> This process is currently manual. There is working being done to fold this into a testing pipeline.
-
-Every merge to chef-server master is built and this clean build is tested
-with all pedant tests. We only run smoke tests for the FIPS mode. Upgrade
-and addon testing must be done in advance of the release.
-
-- [ ] Test as per the matrix from: https://docs.google.com/spreadsheets/d/1_gwxdrMnUiV8t2noi8zs1HyGivtIGMEx9KbYriIw7BY/edit#gid=536218507
-
-- [ ] If this release is being made to address a specific
-  high-urgency, high-severity customer issue or security issue, please
-  *specifically* test that the issue in question is fixed.
-
-If one of these tests has failed, you cannot ship a release until it's fixed.
-Note, no changes other than CHANGELOG/RELEASE_NOTES changes should
-land on master between testing and releasing since we typically tag
-HEAD of master. If something large does land on master, the release
-tag you create should point specifically at the build that you tested.
-The git SHA of the build you are testing can be found in
-`/opt/opscode/version-manifest.json`.
+Every merge to chef-server master is built and this clean build is tested with the automated Integration Test Pipeline https://buildkite.com/chef/chef-chef-server-master-integration-test:
```

Comment:
This happens on every merge, we need to run it through the integration pipeline or ensure the run for the tag that we are shipping was successful.